toolkits/global/packages/global-forms/README.md

```diff
@@ -13,3 +17,265 @@ The `global-forms` component currently uses the `DEFAULT` brand only.
 // Include this with your other components
 @import '@springernature/global-forms/scss/50-components/forms';
 ```
+
+Then you will need to register the Handlebars partials in the `/view` folder. 
```

Comment:
For consistency: 

```suggestion
Then you will need to register the handlebars partials in the `/view` folder. 
```

Reply:
as a long time owner of Sassnotsass.com I had a quick look at the docs for handlebars … and although the logo is in lowercase they seem to use Title Case when it's mentioned throughout the docs. 

(I'm happy either way 😄)